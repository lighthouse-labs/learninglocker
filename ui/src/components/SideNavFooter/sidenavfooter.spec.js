import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import { withInsertCSS } from 'ui/utils/hocs';
import SideNavFooter from './index.js';


const WrappedSideNavFooter = withInsertCSS(SideNavFooter);

test('Renders help link', () => {
  const sideNav = ReactTestRenderer.create(<WrappedSideNavFooter />).toJSON();
  expect(sideNav).toMatchSnapshot();
});
