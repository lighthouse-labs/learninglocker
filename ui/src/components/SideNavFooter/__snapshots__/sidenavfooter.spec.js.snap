// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders help link 1`] = `
<div
  className="sidenav-footer"
>
  <a
    href="https://ht2ltd.zendesk.com/hc/en-us/categories/115000129989-Learning-Locker"
  >
    Help centre
  </a>
  <div>
    Powered by
  </div>
  <a
    href="http://learninglocker.net"
    rel="noopener noreferrer"
    target="_blank"
  >
    Learning Locker Cloud
  </a>
</div>
`;
